tf_files/configs/kube-extract-config.sh

```diff
@@ -72,7 +72,14 @@ revproxyArn=$(g3kubectl get configmaps/global -o=jsonpath='{.data.revproxy_arn}'
 if g3kubectl get secrets/jwt-keys > /dev/null 2>&1; then
   mkdir -p -m 0700 ./jwt-keys
   for keyName in jwt_public_key.pem jwt_private_key.pem; do
-    g3kubectl get secrets/fence-jwt-keys -o=json | jq -r ".data[\"$keyName\"]" | base64 --decode > "./jwt_keys/$keyName"
+    g3kubectl get secrets/fence-jwt-keys -o=json | jq -r ".data[\"$keyName\"]" | base64 --decode > "./jwt-keys/$keyName"
+  done
+fi
+
+if g3kubectl get secrets/fence-private-key > /dev/null 2>&1; then
+  mkdir -p -m 0700 ./ssh-keys
+  for keyName in id_rsa id_rsa.pub; do
+    g3kubectl get secrets/fence-private-key -o=json | jq -r ".data[\"$keyName\"]" | base64 --decode > "./ssh-keys/$keyName"
```

Comment:
Switch to `fence-ssh-keys` - see below ...

Reply:
Fixed at 949c772